app/models/user.rb

```diff
@@ -4,4 +4,10 @@ class User < ApplicationRecord
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :email, presence: true
+
+  scope :personas, -> { where(email: Settings.personas.map(&:email)) }
+
+  def name
```

Comment:
I wonder if we should leave presentational details to the view components and keep them out of the models? Probably fine for now as this is quite small but we've got models littered with presentational details in publish etc.

Reply:
I've created a new model `Persona` that subclasses from `User` and it defines a `default_scope` so we can do `Persona.all`.